python/phylanx/ast/transformation.py

```diff
@@ -232,7 +233,8 @@ def _AugAssign(self, a):
         nn = args[1].__class__.__name__
         if nn == "Add":
             sym = "+"
-        return "store(" + args[0].id + "," + args[0].id + sym + self.recompile(
+        arg0 = '%s' % full_node_name(args[0], args[0].id)
+        return "store(" + arg0 + "," + arg0 + sym + self.recompile(
```

Comment:
The `store()` primitive should be annotated as well.

Reply:
There are more primitive names without annotation in this file ([`slice_column`](https://github.com/STEllAR-GROUP/phylanx/blob/21e0468e6160494f83d1c73615cdfae8ebe951dd/python/phylanx/ast/transformation.py#L120), [`slice`](https://github.com/STEllAR-GROUP/phylanx/blob/21e0468e6160494f83d1c73615cdfae8ebe951dd/python/phylanx/ast/transformation.py#L92), [`shape`](https://github.com/STEllAR-GROUP/phylanx/blob/21e0468e6160494f83d1c73615cdfae8ebe951dd/python/phylanx/ast/transformation.py#L92), etc.)